girder/models/folder.py

```diff
@@ -543,6 +543,39 @@ def parentsToRoot(self, folder, curPath=None, user=None, force=False,
             return self.parentsToRoot(curParentObject, curPath, user=user,
                                       force=force)
 
+    def countItems(self, folder):
+        """
+        Returns the number of items within the given folder.
+        """
+        return self.model('item').find({
```

Comment:
It would be nice to call ``Folder.childItems`` here instead, which would allow virtual dispatch to work more cleanly.